cephfs/statfs.go

```diff
@@ -0,0 +1,76 @@
+package cephfs
+
+/*
+#cgo LDFLAGS: -lcephfs
+#cgo CPPFLAGS: -D_FILE_OFFSET_BITS=64
+#include <stdlib.h>
+#include <cephfs/libcephfs.h>
+*/
+import "C"
+
+import (
+	"fmt"
+	"unsafe"
+)
+
+// CephStatFS instances are returned from the StatFS call. It reports
+// file-system wide statistics.
+type CephStatFS struct {
```

Comment:
How about using [Statfs_t from the syscall package](https://golang.org/pkg/syscall/#Statfs_t) or the one from [golang.org/x/sys/unix](https://godoc.org/golang.org/x/sys/unix#Statfs_t)?

Reply:
It's not the same type. Ceph uses `struct statvfs` not `struct statfs`. They've very similar but not identical.

Reply:
I missed that! Please call the structure `CephStatVFS` in that case. Or is there a good reason to not do so?

Reply:
Sure, no problem with that.

Reply:
All set!